webapp/css/style.css

```
@@ -0,0 +1,198 @@
+  padding: 10px 0; border-top: 1px solid var(--border);
+}
+.row-item:first-child { border-top: none; }
+.pill {
+  display: inline-block; background: var(--bg-raised); border: 1px solid var(--border);
+  border-radius: 99px; font-size: 11px; padding: 1px 8px; color: var(--muted); vertical-align: middle;
+}
+.pill.demo { border-color: var(--yellow); color: var(--yellow); }
+
+.score-pill {
+  border-radius: 99px; font-weight: 800; font-size: 13px; padding: 3px 10px;
+  border: 1px solid currentColor;
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)